data/definitions/npcs.yml

```diff
@@ -72,122 +33,60 @@ chicken:
   hitpoints: 30
   slayer_xp: 3.0
   max_hit_melee: 1
-  style: "stab"
-  examine: "Yep. Definitely a chicken."
-  race: "bird"
-cow_brown:
-  id: 12363
-  copy: cow
-cow_light:
-  id: 12365
-  copy: cow
-cow_calf:
-  id: 12364
+  style: stab
+  examine: Yep. Definitely a chicken.
+  race: bird
+sheep:
+  id: 43
   att: 1
   str: 1
   def: 1
   mage: 1
   range: 1
-  stab_def: -26
-  att_bonus: -20
-  str_bonus: -20
-  slash_def: -26
-  crush_def: -26
-  magic_def: -26
-  range_def: -26
+  stab_def: -10
+  att_bonus: -10
+  str_bonus: -10
+  slash_def: -10
+  crush_def: -10
+  magic_def: -10
+  range_def: -10
   attack_speed: 4
-  wander_radius: 10
   attack_radius: 8
-  hitpoints: 60
-  slayer_xp: 6.0
-  max_hit_melee: 10
-  style: "crush"
-  examine: "Young, but still beefy."
-  race: "cow"
-  height: 15
-cow_calf_light:
-  id: 12366
-  copy: cow_calf
-goblin_mohawk_blue:
-  id: 4262
+  wander_radius: 10
+  hitpoints: 30
+  slayer_xp: 3.0
+  style: stab
+  examine: White and fluffy.
+  race: sheep
+duck_swim:
+  id: 46
+  copy: duck_grey
+  swim: true
+  examine: Quackers.
+rat:
+  id: 47
   att: 1
   str: 1
   def: 1
   mage: 1
   range: 1
-  stab_def: -15
-  att_bonus: -21
-  str_bonus: -15
-  slash_def: -15
-  crush_def: -15
-  magic_def: -15
-  range_def: -15
+  stab_def: -42
+  att_bonus: -47
+  str_bonus: -53
+  slash_def: -42
+  crush_def: -42
+  magic_def: -42
+  range_def: -42
```

Comment:
Any reason why you update this one and not the others?

Reply:
when using the YamlSorter it's done that it switched some things around that one looks to be from duck_grey: id: 6113 or rat: id: 47

Reply:
i've gone through npcs.yml should be fixed and made sure bonus are good